docs/monorepo.md

```diff
@@ -1,77 +1,323 @@
+### 1. Fork and clone the Catalyst monorepo
 
-See [CONTRIBUTING.md](https://github.com/bigcommerce/catalyst/tree/main/CONTRIBUTING.md).
+We'll begin by forking the Catalyst monorepo.
 
-## Getting started from the monorepo
+[You can click here to create a fork](https://github.com/bigcommerce/catalyst/fork) or, navigate to [https://github.com/bigcommerce/catalyst](https://github.com/bigcommerce/catalyst) and click "Fork" at the top of the page.
 
-### Prerequisites
+GitHub will prompt you to choose a name for your fork; for the purposes of this guide, we'll refer to name of the fork as `<YOUR_FORK_NAME>`. You should replace `<YOUR_FORK_NAME>` in all of the commands below with your actual fork name.
 
-* Node.js 20+
-* Corepack-managed `pnpm`
+Back in your terminal, clone your fork:
 
-### Setup
+```shell copy
+git clone git@github.com:<YOUR_GITHUB_USERNAME>/<YOUR_FORK_NAME>.git
+```
 
-1. Clone the project to your local environment:
+Now, change directories into the cloned fork:
 
 ```shell copy
-git clone git@github.com:bigcommerce/catalyst.git && cd catalyst
+cd <YOUR_FORK_NAME>
 ```
 
-2. Use corepack to enable pnpm, then use pnpm to install project dependencies:
+> [!NOTE]
+> Unless otherwise specified, all of the commands that you run in your terminal from this point on should be run from the root of the monorepo repository you just cloned.
+
+### 2. Add a remote pointing to the upstream Catalyst repository
+
+After forking the Catalyst repository, you'll want to set up a connection to the original Catalyst repository so you can easily pull in updates and stay up-to-date with the latest changes. This involves adding the original Catalyst repository as a remote, often named `upstream`.
+
+This will allow you to fetch changes from the main Catalyst project and merge them into your fork as needed.
 
 ```shell copy
-corepack enable pnpm && pnpm install
+git remote add upstream git@github.com:bigcommerce/catalyst.git
 ```
 
-3. Set up environment variables by running:
+To confirm, run the following command in your terminal:
 
 ```shell copy
-cp .env.example .env.local
+git remote -v
+```
+
+You should see output similar to:
+
+```shell
+origin          git@github.com:<YOUR_GITHUB_USERNAME>/<YOUR_FORK_NAME>.git (fetch)
+origin          git@github.com:<YOUR_GITHUB_USERNAME>/<YOUR_FORK_NAME>.git (push)
+upstream        git@github.com:bigcommerce/catalyst.git (fetch)
+upstream        git@github.com:bigcommerce/catalyst.git (push)
 ```
 
-You can find documentation for each field in the `.env.local` file, described in [.env.example](https://github.com/bigcommerce/catalyst/tree/main/.env.example).
+### 3. Enable Corepack
 
-4. If you use VS Code, use the following command to configure VSCode with the project-specific settings the Catalyst team has created:
+Enable Corepack so that you can use `pnpm` as your Node.js package manager with Catalyst. When working in the monorepo, you must use `pnpm`, you cannot use `npm` or `yarn`.
```

Comment:
```suggestion
Enable Corepack so that you can use `pnpm` as your Node.js package manager with Catalyst. When working in the monorepo, you must use `pnpm`, **you cannot use `npm` or `yarn`**.
```